src/windshaft/named-map.js

```diff
@@ -13,13 +14,18 @@ var NamedMap = MapBase.extend({
     //       "2": "/** torque visualization */\n\nMap { ... }"
     //     }
     //   }
-    //
-    json.styles = this._layersCollection.reduce(function (p, c, i) {
-      var style = c.get('cartocss');
+
+    var layers = this._layersCollection.filter(function (layer) {
+      return layer.get('type') !== 'GMapsBase';
```

Comment:
Another idea (💡): instead of adding a new temporary variable (`layers`), we could filter Google Maps base layers inside the `_.reduce` with something like:

```
json.styles = this._layersCollection.reduce(function (memo, layer, index) {
  if (layer.get('type') === 'GMapsBase') { // we could use the helper I mentioned here
    return memo;
  }

  // append styles to memo
  ...
});
```

Or, now that I think about it a bit more... We can probably just get the [layers of type `CartoDB`](https://github.com/CartoDB/cartodb.js/blob/v4/src/geo/map/layers.js#L53-L55) which are the ones with a `cartocss` attribute (not sure if we should also send styles for `torque` layers)...